scripting/include/vscript.inc

```diff
@@ -274,11 +275,18 @@ native HSCRIPT VScript_CompileScript(const char[] script, const char[] id = NULL
  * 
  * @param filepath         Filepath to get a script, 'scripts/vscripts/' are automatically added to the filepath
  * 
- * @return                 HSCRIPT of script, null if could not compile. This MUST be freed when not in use by using HSCRIPT.Release()
+ * @return                 HSCRIPT of script, null if could not compile. This MUST be freed when not in use by using VScript_ReleaseScript().
  * @error                  Invalid filepath.
  */
 native HSCRIPT VScript_CompileScriptFile(const char[] filepath);
 
+/**
+ * Releases a compiled script.
+ * 
+ * @param hscript          HSCRIPT of compiled script
+ */
+native void VScript_ReleaseScript(HSCRIPT hscript);
+
```

Comment:
Would prefer to have methodmap function `HSCRIPT.ReleaseScript()` to match with `HSCRIPT.Release()`